### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

### CODE REVIEW

File: conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt [hunk 1]
```diff
@@ -6,9 +6,6 @@ function errorFunction(int a, float b) {
     int _ = a * b; // @error * is not allowed with operands of different basic types
```
Comment (rdulmina) [inline conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt:6]: This is allowed now. The error here is the result type is `float`. Let's check the other similar places as well

File: conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt [hunk 2]
```diff
@@ -17,7 +14,7 @@ Labels: multiplicative-expr, int, float, decimal
 
 function errorFunction(int a, float b, decimal c) {
     int _ = a * c; // @error * is not allowed with operands of different basic types
-    decimal _ = c / a; // @error / is not allowed with operands of different basic types
```
Comment (rdulmina) [inline conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt:20]: Should we add some set of `output` tests as well to cover allowed combinations

Comment (MaryamZi) [reply conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt:20]: Yes, already created https://github.com/ballerina-platform/ballerina-lang/issues/35826. This PR is to just get the existing tests to pass.

**Review by rdulmina — APPROVED** [review]

File: conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt (hunk 1 above)
Comment (MaryamZi) [reply conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt]: Updated the test in b890cb7.

**Pull request merged**